File: Unidad 3-Funciones/ejercicio2.js
//Escribe CON MODULOS un programa que reciba 2 numeros y te imprima cual es el numero mayor, cual es el numero menor o si son iguales en caso de que sea necesario.
//Autor: RONSSON RAMIRO MAY SANTOS 16/febrero/2109
//Entradas:Dos numeros enteros
/*entradas*/
var a, b;
 a= Number(prompt("Ingresa el 1º numero:"));
b= Number(prompt("Ingresa el 2º numero:"));
compara(a,b);

function compara(a,b){
  if(a===b){
    alert("Los numeros digitados son iguales");
  }else if(a>b){
    alert(+a+" Es el mayor ;");
  } else {
    alert(+b+" Es el mayor ;");
  }
  return;
}
/*
QA
Realizo: Guillermo Canto
Entradas:   Salidas:
5,10        Ok
0,-1        Ok
0,0         Ok
El programa funciona correctamente.
*/

File: Unidad 3-Funciones/Ejercicio2.java
/*
Autor: Pedro Cauich

Problema:Escribe un programa que reciba 2 números y te imprima cual es el número
mayor, cual es el número menor o si son iguales en caso de que sea
necesario. 
 */
import java.util.Scanner;

 public class Main{
     public static void main(String[] args) {
         //Se hace el scan de la entrada, donde el valor de ella se mete al procesamiento, y ese valor resultante entra a la salida
   
        int numeroUno = leerNumero();
        int numeroDos = leerNumero();
        int caso = procesamiento(numeroUno, numeroDos);
        salida(numeroUno, numeroDos, caso); 
     }
     
     public static int leerNumero(){
        Scanner entra1 = new Scanner(System.in); //Declaramos "entra1" como objeto de entrada
        int num1 = entra1.nextInt(); //Leemos el dato, num1 es una variable local
        return num1; //Regresamos el valor de la funcion
     }
     public static int procesamiento(int x, int y){
        if(x>y){
            return 1;
        } else if (y>x){
            return 2;
        } else {
            return 3;
        }
     }
     public static void salida(int a, int b, int c){

         switch (c){
             case 1:
             System.out.println(a + " " + b);
             break;
             case 2:
             System.out.println(b + " " + a);
             break; 
             case 3:
             System.out.println(a + " " + b);
             break; 
         }
     }
 }